deploy-manage/cloud-connect.md

```diff
@@ -0,0 +1,58 @@
+---
+applies_to:
+  deployment:
+    self:
```

Comment:
critical: while product at elastic refers to all self-managed clusters as "self-managed", the docs differentiate them into all of the specific deployment types because they are very different to work with for most features. the `self` tag just is for vanilla, manually installed elastic services. 

is there any restriction on the stack version you have to use for either cloud connected or autoops? 

```suggestion
    self:
    eck:
    ece:
```